src/EnergyPlus/WindowManager.cc

```diff
@@ -3832,11 +3832,9 @@ namespace WindowManager {
             ++iter;
 
             // Calculations based on number of glass layers
-            auto const SELECT_CASE_var(state.dataWindowManager->ngllayer);
-
             GetHeatBalanceEqCoefMatrix(state,
                                        SurfNum,
-                                       SELECT_CASE_var,
+                                       state.dataWindowManager->ngllayer,
```

Comment:
Yes, let's create a SELECT_CASE_var and then just use it as a single argument and that's all.  :upside_down_face: 

OK, I get that it was probably not always used this way, but it is certainly a funny usage that is cleaned up now.